var/spack/repos/builtin/packages/py-ytopt/package.py

```diff
@@ -10,14 +10,20 @@ class PyYtopt(PythonPackage):
     """Ytopt package implements search using Random Forest (SuRF), an autotuning
        search method developed within Y-Tune ECP project."""
 
-    homepage = "https://xgitlab.cels.anl.gov/pbalapra/ytopt"
-    url      = "https://xgitlab.cels.anl.gov/pbalapra/ytopt/raw/release/dist/ytopt-0.1.0.tar.gz"
+    maintainers = ['Kerilk']
 
-    version('0.1.0', sha256='c7081fe3585a5b7a25bcb84733cd2326b72de3bfc4f84d6ad110341f24c3e612')
+    homepage = "https://github.com/ytopt-team/ytopt"
+    url      = "https://github.com/ytopt-team/ytopt/archive/refs/tags/v0.0.1.tar.gz"
 
-    depends_on('py-scikit-learn', type=('build', 'run'))
-    depends_on('py-scikit-optimize', type=('build', 'run'))
+    version('0.0.2', sha256='5a624aa678b976ff6ef867610bafcb0dfd5c8af0d880138ca5d56d3f776e6d71')
+    version('0.0.1', sha256='3ca616922c8e76e73f695a5ddea5dd91b0103eada726185f008343cc5cbd7744')
 
-    def build_args(self, spec, prefix):
-        args = []
-        return args
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-scikit-learn@0.23.1', type=('build', 'run'))
+    depends_on('py-dh-scikit-optimize', type=('build', 'run'))
+    depends_on('py-configspace', type=('build', 'run'))
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-ytopt-autotune@1.1:', type=('build', 'run'))
+    depends_on('py-joblib', type=('build', 'run'))
+    depends_on('py-deap', type=('build', 'run'))
+    depends_on('py-tqdm', type=('build', 'run'))
```

Comment:
Missing a dep on `py-ray+debug`. You'll also need to add that variant to the package.

Reply:
I added the variant and the py-ray dependency. I also added a missing dependency on py-mpi4py which was later added to the required list in a conditional statement. Hopefully what I wrote is correct.

Reply:
Sorry, what I meant was that this package doesn't need a variant, but the `py-ray` package needs a `+debug` variant. The correct dependency should be `depends_on('py-ray+debug', type=('build', 'run'))`

Reply:
HO! my bad. I'll fix this right away.

Reply:
Can you add a `debug` variant to the `py-ray` package?

Reply:
Sure, I wanted to do it early this morning but something came up.

Reply:
Okay, for now I just added the ray variant, based on this:
https://github.com/ray-project/ray/blob/ray-0.8.7/python/setup.py
The debug variant does not bring additional dependencies.
This variant does not exist in more recent versions of the Ray package, so for now I refrained from adding more recent ray releases. This ray package would need some love, there are several variants that are not supported (with many added dependencies, so this is not a trivial change). And those variants change through the releases, for added complexity.

So I kept it simple.

Reply:
Okay, I looked into ray and it seems like `pip install ray[debug]` used to do something, but since https://github.com/ray-project/ray/pull/7031 the additional debug deps are now vendored with the library, so `ray[debug]` is meaningless and was eventually removed as you noted.

So I think the best approach would be to remove the `+debug` variant from `py-ray` and from the `depends_on('py-ray')` since it doesn't mean anything in modern ray.

Sorry for the back and forth!

Reply:
No problem, my Python illiteracy is not helping.
Should be good one the variant removal.